### PR TITLE
uadb almost a full-fledged plugin

### PR DESCRIPTION
**Pull request by m32**
links are converted to taget types
more implemented features from vfs api

unhide pluginmanager module

### CODE REVIEW

File: python/src/python.cpp [hunk 1]
```diff
@@ -383,13 +378,16 @@ XPORT(int, MakeDirectory)(HANDLE hPlugin,const wchar_t **Name,int OpMode) {
     PYTHON_INT(0)
     return result;
 }
+#if 0
+// in far2l only one plugin is allowed at time
```
Comment (elfmz) [inline python/src/python.cpp:382]: what was wrong with this?

Comment (m32) [reply python/src/python.cpp:382]: Currently OpenFilePlugin uses the plugin manager itself, not the plugins installed with it.
It's the same with the analyze api

I still have no idea how to use this api, so I turned it off.

Comment (elfmz) [reply python/src/python.cpp:382]: Yep its same as AnalyseW, but only difference that later is FAR3-specific.
A alternative usage example can tell that NetRocks uses it to 'open' special ini file that describes connection.

Comment (m32) [reply python/src/python.cpp:382]: For now, I will finish this plugin, in a moment there will be another pull anyway - adb is much faster :)

**Pull request closed without merging**